setup.py

```diff
@@ -65,7 +64,10 @@ def get_long_description() -> str:
         "Topic :: Software Development :: Libraries :: Python Modules",
     ],
     install_requires=[
+        "ert",
         "configsuite>=0.6",
+        "ecl",
```

Comment:
Remove `ecl` and `equinor-libres`. They're transitive dependencies of `ert`.

Reply:
`from res.enkf import ErtScript`
`from ecl.rft import EclRFTCell`

and there are many more.

Reply:
I guess it's correct to have them in your `install_requires`, but you can also remove them to have a smaller list of dependencies...

Reply:
I dont think that would be correct, as `setup.py` should reflect reality.